### DISCUSSION TITLE
Read channel content

### DISCUSSION THREAD

**Discussion by adgamerx** [telegraf/telegraf#1993] [Q&A]
how can we read new messages coming in the channel in which the bot is admin?  for forwarding it to another chat to save it as a backup

**Comment by MKRhere** [comment]
Listen for `channel_post`.

```TS
bot.on("channel_post", async ctx => {
	// ...
});
```

You can also use filters. For example, if you want to listen for text messages specifically, you can do:

```TS
import { channelPost } from "telegraf/filters";

bot.on(channelPost("text"), async ctx => {
	// ..
});
```

And so on.
